compiler/resolution/functionResolution.cpp

```diff
@@ -5155,6 +5151,11 @@ static void resolveNew(CallExpr* call) {
             }
 
             resolveExpr(call);
+
+          } else {
+            USR_FATAL(call,
+                      "Sorry, new style initializers don't work with "
+                      "generics yet.  Stay tuned!");
```

Comment:
This change isn't really important, since it will change entirely on my branch, but there isn't much point to removing it from your branch either.